test/powershell/engine/ParameterBinding/Splatting.Tests.ps1

```diff
@@ -0,0 +1,363 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+
+Describe "Hashtable Splatting Parameter Binding Tests" -Tags "CI" {
+
+    BeforeAll {
+        function SimpleTest {
+            param(
+                [Alias('Key')]
+                $Name,
+                $Path
+            )
+
+            "Key: $Name; Path: $Path; Args: $args"
+        }
+    }
+
+    Context "Basic Hashtable Splatting" {
+
+        It "works on cmdlet" {
+            $hash = @{ Verb = "Get"; OutVariable = "zoo" }
+            Get-Verb @hash > $null
+            $zoo | Should -BeOfType 'System.Management.Automation.VerbInfo'
+            $zoo.Verb | Should -BeExactly 'Get'
+        }
+
+        It "works on simple function" {
+            $hash = @{ Name = "Hello"; Blah = "World" }
+            SimpleTest @hash | Should -BeExactly 'Key: Hello; Path: ; Args: -Blah: World'
```

Comment:
Is the Args behavior documented or it is new behavior?

Reply:
It's the existing behavior for simple functions -- any extra argument are stuffed to `$args`.